tests/semanticdb/expect/MethodUsages.scala

```diff
@@ -30,3 +30,9 @@ class MethodUsages {
   m.m18(1)
   m.m18("")
 }
+
+object MethodUsages {
+  var x: Int = 1
```

Comment:
What happens with a val?

Reply:
thats tested in the same file with `m18`

Reply:
i should probably refactor to use the same style